themes/digital.gov/layouts/partials/core/community-guidelines.html

```diff
@@ -0,0 +1,157 @@
+<table class="usa-table usa-table--borderless cop-table usa-table--stacked">
+  <caption>When participating in the community, community members must follow the ground rules for discussions.</caption>
+  <thead>
+    <tr>
+      <th scope="col">
+        <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
+          <use xlink:href="{{ "uswds/img/sprite.svg#check_circle" | relURL }}"></use>
+         </svg>
+          What you should do
+      </th>
+      <th scope="col">
+        <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
+          <use xlink:href="{{ "uswds/img/sprite.svg#cancel" | relURL }}"></use>
+        </svg>
+        What you shouldn't do
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr scope="row">
+      <td>
```

Comment:
Were the stacked data attributes left out intentionally?

Notice the `data-label` on `<th>` and `<td>`.
![image](https://user-images.githubusercontent.com/3385219/224416722-0dab9ce8-1e5a-41c2-b011-ec5cdd2533a2.png)

https://designsystem.digital.gov/components/table/#responsive-stacked-table

Reply:
Mobile view looks even better!

![image](https://user-images.githubusercontent.com/3385219/224426557-2cd85f30-57cf-46dc-bdf6-b576968a2a6a.png)